### DISCUSSION TITLE
standard framerate value of GIFS

### DISCUSSION THREAD

**Discussion by jadaniil** [ImageMagick/ImageMagick#2156] [Help]
If you use the "convert" command and you do convert some JPG images to a .GIF what would be the standard framer ate if you don't use the option "-delay"?

**Comment by snibgo** [comment]
IM has no default. If you don't use "-delay", IM won't write a delay to the GIF.

**Comment by jadaniil** [comment]
But how many frames per second would it have?
--
Отправлено из Mail.ru для Android среда, 17 июня 2020г., 00:19 +02:00 от snibgo  notifications@github.com :

**Reply by fmw42** [reply]
As many as your display device can handle. Often viewers restrict the frame rate and that depends upon the viewer.
